modules/apps/headless/headless-admin-content/headless-admin-content-impl/rest-openapi.yaml

```diff
@@ -360,4 +360,37 @@ paths:
                                 type: array
                     description:
                         ""
+            tags: ["StructuredContent"]
+    "/structured-contents/{structuredContentId}/versions":
+        get:
+            # @review
+            description:
+                Retrieves all versions of a structured content via its ID.
```

Comment:
Do we use ID anywhere else in the yaml descriptions?

Reply:
Yes, we use it in other descriptions :)